Runtime/gestures/mouse_tracking.cs

```diff
@@ -67,39 +92,54 @@ public class MouseTracker {
         public readonly Dictionary<MouseTrackerAnnotation, _TrackedAnnotation> _trackedAnnotations =
             new Dictionary<MouseTrackerAnnotation, _TrackedAnnotation>();
 
-
         public void attachAnnotation(MouseTrackerAnnotation annotation) {
             this._trackedAnnotations[annotation] = new _TrackedAnnotation(annotation);
             this._scheduleMousePositionCheck();
+#if UNITY_EDITOR
```

Comment:
If this is called on an UIWidgetPanel (not an EditorWindow), the _scheduleDragFromEditorMousePositionCheck will still be triggered? maybe we should make sure this wouldn't happen? 

Could we find a way to ensure that all the dragFromEditor relevant logics would never run outside an EditorWindow?